TSOClient/tso.simantics/entities/VMEntity.cs

```diff
@@ -1108,6 +1108,8 @@ private VMPlacementError IndividualUserMovable(VMContext context, bool deleting)
 
         public VMPlacementError IsUserMovable(VMContext context, bool deleting)
         {
+            if (Utils.VMCheatContext.CheatState.TS1_MoveObjects) // for ts1 moveobjects cheat
```

Comment:
This shouldn't be a static variable. Ideally should be similar to other VM state classes, contained within the Context:
```
VMContext {
    VMClock Clock;
    VMArchitecture Architecture;
    VMCheatState Cheats;
}
```

Reply:
Additionally, only cheats that can specficially apply to TS1 should be prefixed with TS1. MoveObjects could potentially apply to both.